<?xml version="1.0" encoding="utf-8"?>

<launch>
  <node pkg="mklc" type="mklc_node" name="mklc_node" output="screen" />
  <node pkg="tf" type="static_transform_publisher" name="central_to_world" args="0 0.9 1.0 -1.57 0 -2.36 world_frame camera_color_frame 100" />
</launch>
